cirq-core/cirq/ops/parallel_gate_test.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import numpy as np
+import sympy
+import cirq
+
+
+@pytest.mark.parametrize(
+    'gate, num_copies, qubits',
+    (
+        (cirq.SingleQubitGate(), 2, cirq.LineQubit.range(2)),
+        (cirq.X ** 0.5, 4, cirq.LineQubit.range(4)),
+    ),
+)
+def test_parallel_gate_operation_init(gate, num_copies, qubits):
+    v = cirq.ParallelGate(gate, num_copies)
+    assert v._sub_gate == gate
+    assert v._num_copies == num_copies
```

Comment:
nit: use public properties `v.sub_gate` and `v.num_copies` instead

Reply:
done